### ISSUE TITLE
md5 severely compromised security

### ISSUE THREAD

**Issue by jukzi** [spotbugs/spotbugs#2155]
Setting up eclipse installer https://www.eclipse.org/setups/installer/?url=https://raw.githubusercontent.com/eclipse-platform/eclipse.platform.releng.aggregator/master/oomph/PlatformSDKConfiguration.setup&show=true
 logs this warning:
`The digest algorithms (md5) used to verify org.eclipse.update.feature,com.github.spotbugs.plugin.eclipse,4.7.2.202208280221-81c9cc3 have severely compromised security. Please report this concern to the artifact provider.`
```

eclipse.buildId=4.25.0.I20220828-1800
java.version=17.0.4
java.vendor=Eclipse Adoptium
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=de_DE
Command-line arguments:  -os win32 -ws win32 -arch x86_64
```
